packages/api/tests/rich-text-detection.test.ts

```diff
@@ -295,10 +295,10 @@ describe('detectFacets', () => {
       const rt = new RichText({ text: input })
       await rt.detectFacets(agent)
 
-      let detectedTags: string[] = []
-      let detectedIndices: { byteStart: number; byteEnd: number }[] = []
+      const detectedTags: string[] = []
+      const detectedIndices: { byteStart: number; byteEnd: number }[] = []
 
-      for (const { facet, ...rest } of rt.segments()) {
+      for (const { facet, ..._rest } of rt.segments()) {
```

Comment:
Oh whoops, leftover from debugging. We can axe this.

```suggestion
      for (const { facet } of rt.segments()) {
```